src/primitives/transaction.cpp

```diff
@@ -13,6 +13,8 @@
 #include <assert.h>
 
 bool g_con_elementsmode = false;
+// TODO: Default to false and override using new chain params class for Sequentia
+bool g_con_sequentiamode = true;
```

Comment:
Rename it to something like "g_con_multi_asset_fees"

Reply:
Agreed about renaming, though I think `g_con_any_asset_fees` would be more accurate. Multi-asset fees makes it sound like it enables transaction fees to be paid for with multiple assets simultaneously, which isn't currently supported and would interfere heavily with Element's blinding mechanism.

Reply:
Done in: https://github.com/SequentiaSEQ/SEQ-Core-Elements/pull/6/commits/0d0ec930a8d190886171a5478b7bc5a589258d96.

Also removed the comment now that the `con_any_asset_flags` flag is documented in chain parameter arguments.